public/components/CampaignList/CampaignList.js

```diff
@@ -11,6 +11,40 @@ class CampaignList extends React.Component {
     campaigns: []
   };
 
+  state = {
+    sortOrder: {
+      'name': true, //false = DESC, true = ASC
+      'type': false,
+      'status': false,
+      'actualValue': false,
+      'startDate': false,
+      'endDate': false
+    },
+    headersCssClasses: {
+      'name': 'campaign-list__header-order--desc name',
```

Comment:
Does any of this need to be stored in the component state? Can it be implied from the redux global state? Same from the classes, can they be constructed on each render, then you don't need to keep them all in sync?